Packs/Base/Scripts/DBotFindSimilarIncidents/DBotFindSimilarIncidents.py

```diff
@@ -463,26 +474,36 @@ def get_score(self):
         Apply transformation for each field in possible transformer
         :return:
         """
+        demisto.debug('DBS: get_score function')
+        demisto.debug(f'DBS: {self.incidents_df=}')
```

Comment:
not sure if pandas does it built-in already or not, but make sure we don't print all the incidents data (might be a lot of data), only the first 5-10 max
also, print the number of rows.